tests/lib/predef_heap/CMakeLists.txt

```diff
@@ -0,0 +1,8 @@
+# SPDX-License-Identifier: Apache-2.0
+
+cmake_minimum_required(VERSION 3.20.0)
+find_package(Zephyr REQUIRED HINTS $ENV{ZEPHYR_BASE})
+project(predef_heap)
+
```

Comment:
Please define C standard project version to avoid having random warnings with whatever compiler end-user have installed on his/her system.